src/home_work_06/task_02_param_dec.py

```diff
@@ -0,0 +1,61 @@
+"""
+Создайте декоратор, который вызывает задекорированную функцию
+пока она не выполнится без исключений
+(но не более n раз - параметр декоратора).
+Если превышено количество попыток,
+должно быть возбуждено исключение типа TooManyErrors
+"""
+
+import all_func
+from functools import wraps
+
+
+class TooManyErrors(Exception):
+    pass
+
+
+def error_count_dec(max_err_count):
```

Comment:
for loop in decorator was expected in this task (call until it would be run w/o errors but no more than..)

Reply:
reworked with for loop